AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -118,6 +117,14 @@ + (void *)userNotificationCenterDelegateContext {
 
 #endif
 
+#pragma mark - MSUserIdContextDelegate
+
+- (void)userIdContext:(MSUserIdContext *)__unused userIdContext didUpdateUserId:(NSString *)userId {
```

Comment:
Why does this method need the `userIdContext` parameter?  Will it be used in the future?

Reply:
I don't actually see where this is being used at all, can you explain?

Reply:
It's actually a convention from Apple on delegates, delegate methods must have the emitter of the delegate as first parameter of the delegate method. I like to think that since delegate methods can be implemented in any classes it also plays the role of prefix for the method to not conflict with other methods of the class itself.